### DISCUSSION TITLE
Problem with proxy hosts.

### DISCUSSION THREAD

**Discussion by destriusz** [NginxProxyManager/nginx-proxy-manager#3455] [Q&A]
Hello.
I have a problem with proxy hosts. I added a rule redirecting my domain to an address in the local network, but it doesn't work.
![obraz](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/105740669/00b51364-4516-4a95-8a54-c79b466d80b6)

**Comment by lastsamurai26** [comment]
If the NPM is in the local network and the domain should be accessible from outside, you must set up port forwarding on your router. 
However, if the NPM is located locally and you are using an external domain, you should host your own DNS so that this domain is not routed to the outside.
